packages/SwingSet/src/controller/controller.js

```diff
@@ -243,16 +243,19 @@ export async function makeSwingsetController(
     // TODO The following assertion may be removed when checkBundle subsumes
     // the responsibility to verify the permanence of a bundle's properties.
     // https://github.com/endojs/endo/issues/1106
-    assert(
+
+    // @ts-ignore `isFake` purposely omitted from type
+    harden.isFake ||
       Object.values(Object.getOwnPropertyDescriptors(bundle)).every(
         descriptor =>
           descriptor.get === undefined &&
           descriptor.writable === false &&
           descriptor.configurable === false &&
           typeof descriptor.value === 'string',
```

Comment:
Should this be relaxed only minimally?
```suggestion
    harden.isFake ||
      Object.values(Object.getOwnPropertyDescriptors(bundle)).every(
        descriptor =>
          typeof descriptor.value === 'string' &&
          descriptor.get === undefined &&
          // @ts-ignore `isFake` purposely omitted from type
          harden.isFake ||
          descriptor.writable === false &&
          descriptor.configurable === false,
```

Reply:
Did you intend to still include the outer `harden.isFake ||` in line 247 of your suggested change above? If so, then the other ones can never fire. But if not...

Could you also revise to put parens in to show how you intended the inside `||` to group compared to the `&&`s? I ask because I'm guessing that you wanted to suppress checking both `.writable` and `.configurable`, but not `.value` or `.get`. Yes?

Reply:
Yes, exactly. Updated.

Reply:
Done